docs/comparison_template_library.md

```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - API
+  - comparisons
+  - Date Comparison
+---
+
+# Documentation for `comparison_template_library` 
+
+The `comparison_template_library` contains pre-made comparisons available for use directly [as described in this topic guide](../topic_guides/customising_comparisons.html#method-1-using-the-comparisonlibrary).
```

Comment:
Would it be possible to add something here (or somewhere) to explain the difference between this template library and the existing comparison library? It might be confusing for users to understand why some comparison appears in one vs the other

Reply:
Yea I am drafting now

Reply:
Created and pushed 😊